document-generator-api/src/main/java/uk/gov/companieshouse/document/generator/api/service/response/ResponseStatus.java

```diff
@@ -4,5 +4,6 @@
 
     CREATED,
     NO_DATA_RETRIEVED,
-    DOCUMENT_NOT_RENDERED
+    DOCUMENT_NOT_RENDERED,
```

Comment:
Should ResponseStatus'es be more generic to match traditional response statuses?

Reply:
do you mean remove Document and just have NOT_RENDERED?